integration_tests/test_coverage.py

```diff
@@ -125,12 +131,12 @@ def test_coverage(profile, no_cleanup):
     if previous_coverage < current_coverage:
         if profile == pytest.profile_ci:
             # In the CI Profile we expect the coverage to be manually updated.
-            assert False, "Coverage is increased from {} to {}. " \
-                          "Please update the coverage in " \
-                          "tests/coverage.".format(
-                previous_coverage,
-                current_coverage
-            )
+            assert False,\
+                "Coverage is increased from {} to {}. "\
+                "Please update the coverage in tests/coverage.".format(
```

Comment:
Since you are updating this, can you also specify the correct path instead of tests/coverage?

Reply:
Updated the path.